Lombiq.ChartJs.Samples/Controllers/SampleController.cs

```diff
@@ -0,0 +1,180 @@
+using Lombiq.ChartJs.Constants;
+using Lombiq.ChartJs.Models;
+using Lombiq.ChartJs.Samples.Constants;
+using Lombiq.ChartJs.Samples.Indexes;
+using Lombiq.ChartJs.Samples.ViewModels.Sample;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json.Linq;
+using OrchardCore;
+using OrchardCore.ContentManagement;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using YesSql;
+
+namespace Lombiq.ChartJs.Samples.Controllers;
+
+public class SampleController : Controller
+{
+    private readonly IOrchardHelper _orchardHelper;
+    private readonly ISession _session;
+    private readonly IContentManager _contentManager;
+
+    public SampleController(
+        IOrchardHelper orchardHelper,
+        ISession session,
+        IContentManager contentManager)
+    {
+        _orchardHelper = orchardHelper;
+        _session = session;
+        _contentManager = contentManager;
+    }
+
+    public async Task<ActionResult> Balance() =>
+        View(new BalanceViewModel
+        {
+            Labels = new[] { Labels.Balance },
+            DataSets = new[]
+            {
+                new ChartJsDataSet
+                {
+                    Label = Labels.Incomes,
+                    BackgroundColor = new[] { "#00aa00" },
+                    Data = new[]
+                    {
+                        (await _session.QueryIndex<IncomePartIndex>()
+                            .ListAsync())
+                            .Where(income => income.Amount is not null)
+                            .Select(income => decimal.ToDouble(income.Amount.Value))
+                            .Sum() as double?,
+                    },
```

Comment:
Would be cleaner like this. [Same below too.](https://github.com/Lombiq/Orchard-Chart.js/pull/21/files#diff-0171dfedf1867d41f0773dddbde4361afa0ad48baa1dab1c355073f6b167f5efR58-R65)
```suggestion
                    Data = new double?[]
                    {
                        (await _session.QueryIndex<IncomePartIndex>()
                            .ListAsync())
                            .Where(income => income.Amount is not null)
                            .Select(income => decimal.ToDouble(income.Amount.Value))
                            .Sum(),
                    },
```